src/components/passphrase-utils.js

```diff
@@ -0,0 +1,76 @@
+const w = window
+
+function randomBytes(length) {
+  const arr = new Uint8Array(length)
+  w.crypto.getRandomValues(arr)
+  return arr.buffer
+}
+
+function fromBufferToB64(buffer) {
+  let binary = ''
+  const bytes = new Uint8Array(buffer)
+  for (let i = 0; i < bytes.byteLength; i++) {
+    binary += String.fromCharCode(bytes[i])
+  }
+  return w.btoa(binary)
+}
+
+// Resolves to a new encryption key, encrypted with
+// the masterKey and ready to be sent to the server on onboarding.
+async function makeEncKey(masterKey) {
+  const subtle = w.crypto.subtle
+  const encKey = randomBytes(64)
+  const iv = randomBytes(16)
+  const impKey = await subtle.importKey(
```

Comment:
How do you develop locally since subtle is not available without https?

Reply:
I have an old Firefox.